modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/display/context/JournalEditArticleDisplayContext.java

```diff
@@ -715,6 +724,29 @@ private boolean _hasInheritedWorkflowDefinitionLink()
 		return false;
 	}
 
+	private boolean _isDDMFormValuesNotEmpty(DDMFormValues ddmFormValues) {
```

Comment:
Hi @marcosapmf , can you put this login inline instead of create this method, please?

Reply:
Hey @jeyvison , due to https://github.com/liferay-data-engine/liferay-portal/pull/91/commits/e4db7ba0e0b0ed04a8b077d81a57d083dfe845d1, this method is no needed anymore :)